examples/src/standalone.html

```diff
@@ -15,10 +15,10 @@ <h1 style="margin-top: 40px;">React Select</h1>
 	</div>
 	<script src="//unpkg.com/react@0.14.2/dist/react.min.js"></script>
 	<script src="//unpkg.com/react-dom@0.14.2/dist/react-dom.min.js"></script>
-	<script src="//unpkg.com/three.js@0.73.0/build/three.min.js"></script>
 	<script src="//unpkg.com/classnames@2.0.0/index.js"></script>
+	<script src="//unpkg.com/prop-types@15.5.10/prop-types"></script>
 	<script src="//unpkg.com/react-input-autosize@0.6.3/dist/react-input-autosize.min.js"></script>
-	<script src="standalone.js"></script>
+	<script src="../../dist/react-select.js"></script>
```

Comment:
Just a note for later: when you clean up the contributing / dev guide, make sure to explain why this file exists and that it depends on the build